### PR TITLE
feat: adjust event filters

### PR DESCRIPTION
**Pull request by phollome**

### CODE REVIEW

File: app/routes/next/explore/events.tsx [hunk 1]
```diff
@@ -74,13 +73,22 @@ export const periodOfTimeValues = [
   "nextMonth",
   "past",
 ] as const;
+const stageValues = ["all", "hybrid", "online", "on-site"] as const;
```
Comment (nac62116) [inline app/routes/next/explore/events.tsx:76]: Should we use the slugs from the database to keep a single source of truth?

And only add the option "all" (which is not present in the db) to the db stages and as default value in the schema?

**Review by nac62116 — CHANGES_REQUESTED** [review]
Nice work 🚀 

Just had one thing. I mentioned it in the comments.

**Comment by nac62116** [comment]
And one more thing. Now that we have vector counts for stage as well, we could display it beneath the radio buttons.

File: app/routes/next/explore/events.tsx (hunk 1 above)
Comment (phollome) [reply app/routes/next/explore/events.tsx]: I've changed this

**Review by nac62116 — APPROVED** [review]
🚢

**Pull request merged**